tests/api/tweens/test_auth.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import pytest
+from pyramid.httpexceptions import HTTPForbidden
+
+from paasta_tools.api.tweens import auth
+
+
+@pytest.fixture
+def mock_auth_tween():
+    with patch.dict(
+        os.environ,
+        {
+            "PAASTA_API_AUTH_ENFORCE": "1",
+            "PAASTA_API_AUTH_ENDPOINT": "http://localhost:31337",
+        },
+    ):
+        with patch("paasta_tools.api.tweens.auth.requests"):
+            yield auth.AuthTweenFactory(MagicMock(), MagicMock())
```

Comment:
not a blocker, but i wonder if these magicmocks should have spec/spec_set set to their respective mockee values